index.html

```diff
@@ -174,7 +173,7 @@ <h3 class="article__title article__title--sidebar">
           <div class="article">
             <img
               class="article__image"
-              src="https://raw.githubusercontent.com/Code-Your-Future/syllabus/master/html-css/assets/news-cycle.jpg"
+              src="/images/edinburgh-damascus_thumbnail.png"
               alt=""
```

Comment:
Need to add in an alt description for the images, important for screen readers

Reply:
thank you for your all encouraging comments Matt, I will add it👍